device_doctor/lib/src/utils.dart

```diff
@@ -7,6 +7,17 @@ import 'dart:io';
 import 'package:logging/logging.dart';
 import 'package:path/path.dart' as path;
 
+/// Properties used in device_doctor tool.
```

Comment:
Can we document that this tool requires adb and idevice_id and remove this complexity?

Reply:
Removed.
I will document this in a following PR which focus on the REAMED.